assets/launcher/lang/sklmessages_vi_VN.properties

```diff
@@ -69,7 +69,7 @@ editor.compatibility=Chế độ tương thích
 editor.compatibility.subtext=Tùy chọn này tắt skin.
 editor.compatibility.sublink=Xem thêm
 editor.compatibility.alert.title=Chế độ tương thích
-editor.compatibility.alert.text=TTùy chọn này là cần thiết cho một vài bản modpack, nhưng hầu như thì không. Chỉ bật tùy chọn này khi bạn biết bạn đang làm gì!
+editor.compatibility.alert.text=Tùy chọn này là cần thiết cho một vài bản modpack, nhưng hầu như thì không. Chỉ bật tùy chọn này khi bạn biết bạn đang làm gì!
```

Comment:
@TrueEL I thought the second "T" is a leftover from English "This", so I've removed it. If I'm wrong, please, let me know :)

Reply:
Yea, remove it for me :) I did leave some leftover. Thanks for finding that out!